source/tutorials/first-steps/nix-language.md

```diff
@@ -953,7 +953,7 @@ Example:
 /nix/var/nix/profiles/per-user/root/channels/nixpkgs
 ```
 
-The value of a named path is a file system path that depends on the contents of the [`$NIX_PATH`][NIX_PATH] environment variable.
+The value of a [lookup path](https://nixos.org/manual/nix/unstable/language/lookup-path) is a file system path that depends on the value of [`builtins.nixPath`](https://nixos.org/manual/nix/stable/language/builtin-constants#builtins-nixPath).
```

Comment:
I think it would be good to describe the relation between $NIX_PATH and builtins.nixPath . either here or in the nix manual

Reply:
It's being worked on heavily in the Nix manual, this is why I added this here in advance. Enough of it should land in the next release.

Reply:
if you mean https://github.com/NixOS/nix/pull/9114 i don't think it clearly states how builtins.nixPath relates to  $NIX_PATH.
a formulation like: 
builtins.nixPath returns the value of $NIX_PATH as a list of attribute sets.

Reply:
No, this: https://github.com/NixOS/nix/pull/9113